pulp_2_tests/tests/rpm/api_v2/test_content_applicability.py

```diff
@@ -83,7 +87,12 @@
         }
     }
 }
-"""A schema for a content applicability report for a consumer."""
+"""A schema for a content applicability report for a consumer.
```

Comment:
@kersommoura see if this gives enough information or if you think there should be any additional information. Thanks!